opam/descr

```diff
@@ -1 +0,0 @@
-Connect to remote XenServer VM consoles
```

Comment:
Opam supports having the description in `vncproxy.descr` on the root folder, next to the `opam` file. We are not doing it in other packages, so it's fine to have the file removed here